praw/models/reddit/redditor.py

```diff
@@ -93,3 +121,51 @@ def unblock(self):
     def unfriend(self):
         """Unfriend the Redditor."""
         self._friend(method='DELETE', data={'id': str(self)})
+
+
+class UserStream(object):
```

Comment:
For consistency, can we rename this `RedditorStream` and use `redditor` in place of `user` elsewhere?